### DISCUSSION TITLE
Fibonacci Python

### DISCUSSION THREAD

**Discussion by RichCoastCR** [community/community#146703] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Any good Fibonacci repos in Python?

**Comment by arcesoftware** [comment, marked as answer]
Check this one out: 
https://github.com/arcesoftware/Fibonacci-Fast-Fourier-Transform

**Reply by RichCoastCR** [reply]
Thank you this is amazing!

**Comment by RichCoastCR** [comment]
This is all I was looking for.

**Comment by RichCoastCR** [comment]
Check the link from arcesoftware is very good!
